tests/test_approval_modules_snapshots/example_pump_asset_hierarchy.yaml

```diff
@@ -74,7 +74,7 @@ Transformation:
     \  PumpNumber,\n  PumpHP,\n  HighHeadShutOff,\n  DesignPointHeadFT,\n  DesignPointFlowGPM,\n\
     \  LowHeadFT,\n  LowHeadFlowGPM,\n  PumpControl,\n  PumpModel,\n  Shape__Length,\n\
     \  Enabled,\n  DesignPointHeadFT,\n  LowHeadFT,\n  FacilityID,\n  InstallDate,\n\
-    \  LifeCycleStatus,\n  LocationDescription\n  ) as metadata\nfrom pump_assets.`collections_pump`\n"
+    \  LifeCycleStatus,\n  LocationDescription\n  ) as metadata\nfrom `{{raw_db}`.`collections_pump`\n"
```

Comment:
This is unexpected? It looks like the {{raw_db}} variable in  `example_pump_asset_hierarchy` is not correctly replaced.

Reply:
Yes, typo. Will fix.